Sources/EdgeBridgeConstants.swift

```diff
@@ -25,4 +25,24 @@ enum EdgeBridgeConstants {
     enum JsonValues {
         static let EVENT_TYPE = "analytics.track"
     }
+
+    enum AnalyticsKeys {
+        static let ACTION = "action"
+        static let STATE = "state"
+        static let CONTEXT_DATA = "contextdata"
+    }
+
+    enum EdgeKeys {
```

Comment:
I got a little confused on the naming here since this is the Analytics format. renaming suggestion:
AnalyticsKeys -> MobileCoreKeys
EdgeKeys/Values -> AnalyticsKeys/Values